src/test/java/com/cronutils/model/time/ExecutionTimeQuartzWithDayOfYearExtensionIntegrationTest.java

```diff
@@ -43,47 +42,52 @@ public void setUp() throws Exception {
     @Test
     public void testForCron() {
         assertEquals(ExecutionTime.class, ExecutionTime.forCron(parser.parse(BI_WEEKLY_STARTING_WITH_FIRST_DAY_OF_YEAR)).getClass());
-        assertEquals(ExecutionTime.class, ExecutionTime.forCron(parser.parse(FIRST_QUATER_BI_WEEKLY_STARTING_WITH_FIRST_DAY_OF_YEAR)).getClass());
+        assertEquals(ExecutionTime.class, ExecutionTime.forCron(parser.parse(FIRST_QUARTER_BI_WEEKLY_STARTING_WITH_FIRST_DAY_OF_YEAR)).getClass());
         assertEquals(ExecutionTime.class, ExecutionTime.forCron(parser.parse(WITHOUT_DAY_OF_YEAR)).getClass());
         assertEquals(ExecutionTime.class, ExecutionTime.forCron(parser.parse(WITHOUT_SPECIFIC_DAY_OF_YEAR)).getClass());
     }
 
     @Test
     public void testNextExecutionEveryTwoWeeksStartingWithFirstDayOfYear() {
-        final ZonedDateTime now = truncateToDays(ZonedDateTime.now().minusDays(1));
-        final int dayOfYear = now.getDayOfYear();
-        final int dayOfMostRecentPeriod = dayOfYear % 14;
-        final ZonedDateTime expected = now.plusDays(15 - dayOfMostRecentPeriod);
         final ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(BI_WEEKLY_STARTING_WITH_FIRST_DAY_OF_YEAR));
-        assertEquals(expected, executionTime.nextExecution(now).get());
+
+        for (int i = 1; i < 30; i++) {
+            final ZonedDateTime now = truncateToDays(ZonedDateTime.of(2017, 10, i, 0, 0, 0, 0, UTC));
+            final int dayOfMostRecentPeriod = (now.getDayOfYear() - 1) % 14;
+            final ZonedDateTime expected = now.plusDays(14 - dayOfMostRecentPeriod);
+            assertEquals("Wrong next time from " + now, expected, executionTime.nextExecution(now).get());
+        }
     }
 
     @Test
     public void testNextExecutionEveryTwoWeeksStartingWithFirstDayOfYearIssue249() {
```

Comment:
Is this test really necessary? The test above does the same thing or am I missing something?

Reply:
True, I guess it does. Was a bit in a solution mode, so didn't reflect on that.

Reply:
I remove the duplicated test case